Automattic/pocket-casts-android
# Add source to podcast details analytics

## Description

This PR adds source property to `podcast_screen_shown` analytics event.

I also sorted `SourceView` entries alphabetically.

## Testing Instructions

Open podcast details from different entry points. You should see `source` property for `podcast_screen_shown` analytics event.

```
🔵 Tracked: podcast_screen_shown, Properties: {"source":"player","has_dynamic_font_size":true,"user_is_logged_in":true,"plus_has_subscription":true,"plus_has_lifetime":false,"plus_subscription_type":"plus","plus_subscription_tier":"patron","plus_subscription_platform":"gift","plus_subscription_frequency":"none","platform":"phone"}
🔵 Tracked: podcast_screen_shown, Properties: {"source":"podcast_list","has_dynamic_font_size":true,"user_is_logged_in":true,"plus_has_subscription":true,"plus_has_lifetime":false,"plus_subscription_type":"plus","plus_subscription_tier":"patron","plus_subscription_platform":"gift","plus_subscription_frequency":"none","platform":"phone"}
🔵 Tracked: podcast_screen_shown, Properties: {"source":"discover","has_dynamic_font_size":true,"user_is_logged_in":true,"plus_has_subscription":true,"plus_has_lifetime":false,"plus_subscription_type":"plus","plus_subscription_tier":"patron","plus_subscription_platform":"gift","plus_subscription_frequency":"none","platform":"phone"}
🔵 Tracked: podcast_screen_shown, Properties: {"source":"episode_details","has_dynamic_font_size":true,"user_is_logged_in":true,"plus_has_subscription":true,"plus_has_lifetime":false,"plus_subscription_type":"plus","plus_subscription_tier":"patron","plus_subscription_platform":"gift","plus_subscription_frequency":"none","platform":"phone"}
```

## Checklist
 
#### I have tested any UI changes...
